ui/widgets/nuxeo-user-tag.js

```diff
@@ -56,8 +56,17 @@ import './nuxeo-tooltip.js';
           a:hover {
             @apply --nuxeo-link-hover;
           }
+
+          .user-tag {
+            white-space: nowrap;
+            overflow: hidden;
+            text-overflow: ellipsis;
+          }
+          .nuxeo-tag {
+            width: 100%;
+          }
         </style>
-        <nuxeo-tag>
+        <nuxeo-tag class="nuxeo-tag">
```

Comment:
I don't think we need to create a new class for this, since we can simply use (and are already using) the tag name.